.github/workflows/release.yml

```diff
@@ -1,14 +1,13 @@
-name: Test Release
+name: Release
 
 on:
-  pull_request:
   push:
-    branches:
-      - staging
+    tags:
```

Comment:
add the check for `branches` here as well? only push from `develop`? otherwise all tags even if accidentally pushed to some other branch may trigger a release.